site/styles/_meta-sub-item.scss

```diff
@@ -80,27 +100,4 @@
       /* stylelint-enable selector-max-compound-selectors */
     }
   }
-
-  /* stylelint-disable */
-  @include media-breakpoint-up(sm) {
-    grid-template-columns: 175px minmax(100px, 1fr);
-    grid-template-rows: auto auto minmax(0, 1fr);
-
-    .synopsis {
-      grid-column: auto;
-      grid-row: 2;
-      margin-bottom: 10px;
-    }
-    .poster-container {
-      grid-row: 1 / 4;
-    }
-    .runtime {
```

Comment:
looks like you removed the .runtime styles but i cant see them reinserted?

Reply:
Yeah the runtime element has no margin so there is no need to override margins to 0 at different breakpoints.